pkg/crc/preflight/preflight_darwin.go

```diff
@@ -50,18 +52,22 @@ var traySetupChecks = [...]PreflightCheck{
 		flags:            SetupOnly,
 	},
 	{
-		checkDescription: "Checking if launchd configuration for daemon exists",
-		check:            checkIfDaemonPlistFileExists,
-		fixDescription:   "Creating launchd configuration for daemon",
-		fix:              fixDaemonPlistFileExists,
-		flags:            SetupOnly,
+		checkDescription:   "Checking if launchd configuration for daemon exists",
+		check:              checkIfDaemonPlistFileExists,
+		fixDescription:     "Creating launchd configuration for daemon",
+		fix:                fixDaemonPlistFileExists,
+		flags:              SetupOnly,
+		cleanupDescription: "Removing launchd configuration for daemon",
```

Comment:
This is consistent with the other messages, but maybe they should all be changed to  `CodeReady Containers daemon`